README.md

```diff
@@ -1023,6 +1024,53 @@ export default class MyComponent extends Vue {
 </script>
 ```
 
+### 8.6 Vuex
+
+Basically, vuex is like a "middleware" it is responsible for being the intermediary between the API and the Components.
+
+`API => Vuex => Components`
+
+![Modelo](./assets/image/modelos.png)
+
+#### VUEX attributes
+
+- state:{} - controls the state;
+- mutations:{} - change the state;
+- actions:{} - actions within a given context;
+- modules:{} - single divisions that contain `states, mutations and actions` watch out for certain modules
+
+#### Logical order of vuex
+
+![Modulo](./assets/image/logic_order.png)
+
+An **ACTION** must call a **MUTATION** to change a **STATE**;
+
+- **mutations** are synchronous functions, that is, when the system is triggered, it has to wait for the mutation to finish.
+
+- **action** are asynchronous, `async` functions that can be executed and don't need to wait for them to finish.
+
+It is not mandatory to use action together with mutation. Action is used when you need a promise, usually when you make a request to the API or any task that requires the use of a promise.
+
+VUE 2: $store -> access to vuex.
+VUE 3: store -> access to vuex.
+
+anything that changes in vuex will be `'replicated'` in components.
+
+to call an **action** use `store.dispatch`
+to call a **mutation** use `store.commit`
+
+An action receives as a parameter a `context` and a `payload`.
+
+A mutation takes a `state` and a `payload` as a parameter.
```

Comment:
Question: I do not remember the idea behind this PR, but I'm missing Vuex code snippet examples.  Also, I would not put thoses snippet as we do not use it this way:

```JS
`store.dispatch`
`store.commit`
```